policyengine_us/reforms/reforms.py

```diff
@@ -1,5 +1,20 @@
+from .congress.delauro import create_american_family_act_with_baby_bonus_reform
 from .winship import create_eitc_winship_reform
+from policyengine_core.reforms import Reform
+import warnings
 
 
 def create_structural_reforms_from_parameters(parameters, period):
-    return create_eitc_winship_reform(parameters, period)
+    afa_reform = create_american_family_act_with_baby_bonus_reform(
+        parameters, period
+    )
+    winship_reform = create_eitc_winship_reform(parameters, period)
+
+    if afa_reform is not None:
+        if winship_reform is not None:
+            warnings.warn(
```

Comment:
@nikhilwoodruff how can we test this code?

Reply:
OK we should strive for 100% code coverage but merging for now and filing https://github.com/PolicyEngine/policyengine-us/issues/2630